Nexd/Nexd/ui/onboarding/RegistrationViewController.swift

```diff
@@ -21,7 +21,9 @@ class RegistrationViewController: ViewController<RegistrationViewController.View
     private var keyboardObserver: KeyboardObserver?
     private var keyboardDismisser: KeyboardDismisser?
     lazy var logo = UIImageView()
-
+    private let caShapeLayer = CAShapeLayer()
+    private var didAgreeTermsOfUse: Bool = false
```

Comment:
I think this should typically stored in the viewmodel

Reply:
ah I think this screen doesn't have a view model yet... someone else worked on it while I created VMs for the remaining parts of the app... - and maybe in a future change we should refactor this screen to SwiftUI (to hopefully get rid of the weird scrolling behavior when the keyboard is shown/dismissed)...

So until then I guess it's ok to keep the variable just here to avoid the effort of creating a view model which we would have to refactor later on anyways..